src/app/components/Persist/DeleteProfileButton.tsx

```diff
@@ -33,7 +33,7 @@ export function DeleteProfileButton({ prominent }: DeleteProfileButtonProps) {
     <>
       <Button
         color="status-error"
-        label={t('persist.loginToProfile.deleteProfile.button', 'Delete profile')}
+        label={t('persist.loginToProfile.deleteProfile.button', 'Forgot password?')}
```

Comment:
Can we take into account the second context where button is used? When user has an open profile we will show "Forget password?" in Profile tab.
![Screenshot from 2024-04-05 09-06-38](https://github.com/oasisprotocol/oasis-wallet-web/assets/891392/787372a6-8b0d-4ea3-90d5-b548d37974a8)

Reply:
ah :/

Reply:
Checking this variant with Don: 
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/b97736ee-9c8d-49c5-bc4e-5038ef6d8766)